config/i18n.json

```diff
@@ -1125,10 +1125,10 @@
     "IndexedDBStorage": "Local Browser Storage",
     "LearnMore": "Learn more about DIM Sync",
     "MenuTitle": "Sync & Backups",
-    "ProfileErrorTitle": "DIM Sync Error",
-    "ProfileErrorBody": "We had a problem loading your data from DIM Sync. We'll keep trying to load it automatically.",
-    "UpdateErrorTitle": "DIM Sync Error",
-    "UpdateErrorBody": "We had a problem saving your data to DIM Sync. We'll automatically try again later.",
+    "ProfileErrorTitle": "DIM Sync Download Error",
+    "ProfileErrorBody": "We had a problem loading your latest settings, tags, loadouts, and searches from DIM Sync. Your data is still on our servers, and any updates you make locally will be saved when we can reconnect. We'll keep trying to load your data in the background as you use DIM.",
+    "UpdateErrorTitle": "DIM Sync Save Error",
+    "UpdateErrorBody": "We had a problem saving your data to DIM Sync. We'll automatically try again as you use DIM.",
```

Comment:
i think the above message's "keep trying" is very reassuring compared to "automatically try again"
"as you use DIM" is both true and specific but not sure if it buys any useful user understanding.
if the goal is to emphasize that the loaded client is what will do the action, maybe "DIM will keep retrying while it's open" or "DIM will keep retrying while it's in use."?